email-spring-boot-starter/src/main/java/io/_57blocks/email/EmailServiceImpl.java

```diff
@@ -116,27 +113,27 @@ private void processAttachments(MimeMessageHelper messageHelper, Attachment[] at
     }
   }
 
-  private String getSubjectTemplate(String template, String contentPattern,
-      String subjectPattern) {
-    contentPattern = contentPattern.substring(0, contentPattern.indexOf('/'));
-    subjectPattern = subjectPattern.substring(0, subjectPattern.indexOf('/'));
 
-    return template.replace(contentPattern, subjectPattern);
-  }
-
-  private String getHtmlSubjectTemplate(String template) {
+  private String getSubjectTemplate(String template) {
 
-    String htmlPattern = properties.getTemplate().getHtml().getPattern();
     String subjectPattern = properties.getTemplate().getSubject().getPattern();
-
-    return getSubjectTemplate(template, htmlPattern, subjectPattern);
+    return concatWithPrefix(subjectPattern, template);
   }
 
-  private String getTextSubjectTemplate(String template) {
+  private String getContentTemplate(String template, boolean isHtml) {
```

Comment:
Might be better to use enum than `isHtml`, we might support editable html email later.

Reply:
there is only 2 types of email: text and html. To implement editable html email, it only needs to add `message.addInline(resource, type)` to attach the resources into html content.

Reply:
ok